projects/packages/my-jetpack/_inc/components/product-cards-section/protect-card/protect-value-section.tsx

```diff
@@ -0,0 +1,113 @@
+import { Gridicon } from '@automattic/jetpack-components';
+import { Popover } from '@wordpress/components';
+import { useViewportMatch } from '@wordpress/compose';
+import { __, sprintf } from '@wordpress/i18n';
+import { useState } from 'react';
+import { PRODUCT_STATUSES } from '../../../constants';
+import useProduct from '../../../data/products/use-product';
+import { getMyJetpackWindowInitialState } from '../../../data/utils/get-my-jetpack-window-state';
+import { timeSince } from '../../../utils/time-since';
+
+import './style.scss';
+
+const ProtectValueSection: React.FunctionComponent = () => {
+	const slug = 'protect';
+	const { detail } = useProduct( slug );
+	const { status } = detail;
```

Comment:
Might want to add something like this, just in case something weird happens and the detail is null instead of an object
```suggestion
	const { status = PRODUCT_STATUSES.INACTIVE } = detail || {};
```